src/fides/api/api/v1/endpoints/system.py

```diff
@@ -366,13 +373,47 @@ async def create(
             scopes=[SYSTEM_READ],
         )
     ],
-    response_model=List[BasicSystemResponse],
+    response_model=Union[List[BasicSystemResponse], Page[BasicSystemResponse]],
     name="List",
```

Comment:
I think it'd be nice to include that this is optionally paginated like you did for the dataset endpoint
```suggestion
    name="List systems (optionally paginated)",
```